linux/check_os_fullname/miracle_fullname_map.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Guest id "asianux8_64Guest" and full name "MIRACLE LINUX 8 (64-bit)" is available on ESXi 6.7P06 or later
+
+- name: "Set fact of expected guest id on ESXi {{ esxi_version }}"
+  ansible.builtin.set_fact:
+    expected_guest_major_ver: "{{ [guest_os_ansible_distribution_major_ver | int, 8 ] | min }}"
+
+- name: "Set facts when vmtools_version is larger than 12.3.5"
+  when:
+    - vmtools_version is defined
+    - vmtools_version is version('12.3.5', '>')
+  block:
+    - name: "Set fact of guest OS is not other Linux"
+      ansible.builtin.set_fact:
+        guest_is_otherlinux: false
+
+    - name: "Set fact of expected guest id suffix"
+      ansible.builtin.set_fact:
+        expected_guest_id_suffix: "{{ expected_guest_id_suffix | replace('64', '_64') }}"
```

Comment:
`guest_is_otherlinux` and `expected_guest_id_suffix` can be put in one set_fact task.